### ISSUE TITLE
`kamal setup` fails with `ArgumentError: No database file specified. Missing argument: database`

### ISSUE THREAD

**Issue by benjaurr** [basecamp/kamal#932]
When trying to setup a rails server, independent of the database adapter I choose to run (I've tried with sqlite3, mysql and postgres), the healthcheck fails with `ArgumentError: No database file specified. Missing argument: database (ArgumentError)`. 

When looking at rails source code, this is in the [sqlite3 adapter code](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb), so I do not understand why it would fail when using other adapters

Here's a copy of my `database.yml` and `deploy.yml`:

```yml
default: &default
  adapter: sqlite3
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
  timeout: 5000

development:
  <<: *default
  database: storage/development.sqlite3

test:
  <<: *default
  database: storage/test.sqlite3

production:
  <<: *default
  database: storage/production.sqlite3
```

```yml
service: x

image: x

# Deploy to these servers.
servers:
  - x

volumes:
  - "/db:/rails/sqlite"
  - "/storage:/rails/storage"

registry:
  username: x

  password:
    - KAMAL_REGISTRY_PASSWORD

env:
  secret:
    - RAILS_MASTER_KEY
``

**Comment by dhh** [comment]
Sounds like maybe you're not using the default bin/docker-entrypoint that runs `rails db:prepare` to ensure that the sqlite3 database is created if missing. You can generate a new Rails app to see what it should look like and how it's used in the default Dockerfile.

**Comment by benjaurr** [comment]
Thanks for the input. Just to give closure to the issue: It was an issue with the builder's context. I noticed it when connecting to the server and running `cat config/database.yml` and the database file was not the same as it was in my local setup. The [docs have a section on the context](https://kamal-deploy.org/docs/configuration/builders/#build-context), and since I was working on uncommited changes, I updated the deploy.yml to include:

```
builder:
  context: .
```

That fixed my problem. I apologize for the non-issue
